docs/source/_templates/searchbox.html

```diff
@@ -0,0 +1,14 @@
+{%- if builder != 'singlehtml' %}
+
+{% if theme_pytorch_project == 'tutorials' %}
+  {% set search_project = 'Tutorials' %}
+{% else %}
+  {% set search_project = 'Docs' %}
+{% endif %}
+
+<div role="search">
+  <form id="rtd-search-form" class="wy-form">
+    <input type="text" placeholder="Search {{ search_project }}" />
+  </form>
+</div>
+{%- endif %}
```

Comment:
copied from `searchbox.html` of `pytorch_sphinx_theme`, removed unnecessary lines.